pipelines/HSC/DRP-RC2_subset.yaml

```diff
@@ -158,14 +158,7 @@ tasks:
   objectTableCoreRefCatMatchWholeSkyPlot:
```

Comment:
you can remove all the configs expect "band" for matchedVisitCoreWholeSkyPlot and objectTableCoreWholeSkyPlot

Reply:
you can also do this in pipelines/HSC/DRP-ci_hsc.yaml